app/models/user.rb

```diff
@@ -1,2 +1,9 @@
 class User < ApplicationRecord
+  has_many :posts, class_name: 'Post', foreign_key: 'author_id'
```

Comment:
- [ ] Please remove the `class_name` method from the associations because it's not adding anything! From the start, the association says that a `User` has many `Posts` so the table is already called `posts` so it doesn't make sense if we tell the model the class name of it again. Instead, this method is used when a model has an association with another model but in a different name. For ex:
`User has many articles, class_name: 'Post'`

> Please apply the changes in all similar situations through the associations

Reply:
Thank you for the feedback.